### ISSUE TITLE
.deb package for the 2.0 doesn't install on Kubuntu 22.04 / using QApt

### ISSUE THREAD

**Issue by DesiOtaku** [f3d-app/f3d#751]
I downloaded https://github.com/f3d-app/f3d/releases/download/v2.0.0/F3D-2.0.0-Linux-x86_64.deb on Kubuntu 22.04 and QApt can't seem to figure how how to install it. I haven't tried to install it via dpkg yet but I just wanted to report that QApt doesn't like the current package as it is.

**Comment by mwestphal** [comment]
HI @DesiOtaku 

So I just tried on a fresh Kubuntu and i do **reproduce** the issue, however:
1. `sudo apt install ./F3D-2.0.0..." works perefectly
2. `sudo dpkf -i ./F3D-2.0.0..." works perfectly
3. By double clicking on the file, I can open it with "Discover" which works perfectly as well
4. Running `qapt-deb-installer` from the command line show the same issue but do not provide any information about why it does not want to install.
5. qapt-batch-install does not support installing a local .deb file

At this poins I do not see much more I can do, this looks like some kind of issue or inconsistencies in qapt that we trigger involuntarly.

**Comment by mwestphal** [comment]
The .deb package should be improved though, with the target to install in /opt and just put only the necessary bit in /usr, similar to what is done here: https://github.com/f3d-app/f3d-bin-PKGBUILD

**Comment by v0id-strike** [comment]
is this still valid bug to work on it?

**Comment by mwestphal** [comment]
> is this still valid bug to work on it?

I think so, one would need to check that 3.0 release has the same issue, but nothing changed on that regard afaik.
That being said, I have no way forward to improve the problem, we have not even been able to understand if this is an issue in F3D .deb package or in qapt-deb-installer.

Let me know If I should assign it to you @v0id-strike

**Comment by v0id-strike** [comment]
Let me download kubuntu, will let you know once i figure it out what is happening

**Comment by v0id-strike** [comment]
Nope, i don't have any idea. I read through log. didn't find anything

**Comment by DesiOtaku** [comment]
![Image](https://github.com/user-attachments/assets/84faaa27-98ed-426e-ae6e-5507ae1e44b2)

I decided to test it out with the 3.0 release and it gives the same error.
